### PR TITLE
Add Paw-Moya in CommunityProjects.md

### PR DESCRIPTION
**Pull request by narlei**
Add documentation for Paw Extension to export requests to Moya files.

Extension in Paw site:
https://paw.cloud/extensions/SwiftMoyaCodeGenerator

### CODE REVIEW

File: docs/CommunityProjects.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ We appreciate all the work being done by the community around Moya. If you would
 - [Moya-EVReflection](https://github.com/evermeer/EVReflection/tree/master/Source/Alamofire/Moya) - EVReflection bindings for Moya for easier JSON serialization (including subspecs for [RxSwift](https://github.com/evermeer/EVReflection/tree/master/Source/Alamofire/Moya/RxSwift) and [ReactiveCocoa](https://github.com/evermeer/EVReflection/tree/master/Source/Alamofire/Moya/ReactiveCocoa))
 - [Moya-Marshal](https://github.com/JARMourato/Moya-Marshal) - Marshal bindings for Moya for easier JSON serialization
 - [Moya-Decodable](https://github.com/xiaoyaogaojian/Moya-Decodable) - Decodable bindings for Moya for easier JSON serialization
+- [Moya-Paw](https://github.com/narlei/SwiftMoyaCodeGenerator) - Extension for [Paw Mac App](https://paw.cloud) to generate Moya files based in rest documentation.
```
Comment (sunshinejr) [inline docs/CommunityProjects.md]: It's a trivial one, but can we change the name to `SwiftMoyaCodeGenerator` instead of `Moya-Paw`? 😉

Comment (narlei) [reply docs/CommunityProjects.md]: Done, thank you for your tip.

**Comment by ashfurrow** [comment]
Thanks a lot for contributing to Moya! I've invited you to join the Moya GitHub organization – no pressure to accept! If you'd like more information on what that means, check out our [contributor guidelines](https://github.com/Moya/contributors) and feel free to reach out with any questions.

**Comment by narlei** [comment]
@ashfurrow thank's for the invite. Accepted just now! It's a pleasure be a part of Moya project.

**Pull request merged**